src/forms/textInput.test.jsx

```diff
@@ -10,6 +11,16 @@ describe('TextInput', function() {
 		MAX_LEN = '20',
 		ERROR_TEXT = 'Too wimpy.';
 
+	// Note: since TextInput is a functional component
+	// we need to wrap it in a stateful component to use TestUtils effectively
+	const TestWrapper = createReactClass({
```

Comment:
might be good to make this a utility as we move other MWC components to stateless. i suggest in `testUtils`